src/Infrastructure/LeanCode.ForceUpdate/LeanCode.ForceUpdate.Services/CQRS/VersionSupportQH.cs

```diff
@@ -26,7 +26,7 @@ VersionHandler versionHandler
 
     public async Task<VersionSupportDTO?> ExecuteAsync(HttpContext context, VersionSupport query)
     {
-        if (!Version.TryParse(query.Version, out var version) || !Enum.IsDefined<PlatformDTO>(query.Platform))
+        if (!Version.TryParse(query.Version, out var version) || !Enum.IsDefined(query.Platform))
```

Comment:
Is is necessary? Won't it resolve the `IsDefined` that we want?

Reply:
I actually removed `<PlatformDTO>` from `IsDefined` or are asking if `Enum.IsDefined` is necessary?

Reply:
Ah, gosh, I mistook the colors...